qa/rpc-tests/feature_fedpeg.py

```diff
@@ -143,14 +183,15 @@ def run_test(self):
             pegtxid = sidechain.claimpegin(raw, proof)
             raise Exception("Peg-in should not be mature enough yet, need another block.")
         except JSONRPCException as e:
+            print('ERROR:', e.error)
             assert("Peg-in Bitcoin transaction needs more confirmations to be sent." in e.error["message"])
 
         # Should fail due to non-witness
         try:
-            pegtxid = sidechain.claimpegin(raw, proof, get_new_unconfidential_address(parent))
+            pegtxid = sidechain.claimpegin(raw, proof, get_new_unconfidential_address(sidechain))
             raise Exception("Peg-in with non-matching claim_script should fail.")
         except JSONRPCException as e:
-            print(e.error["message"])
+            print('ERROR:', e.error)
```

Comment:
Are you sure removing ['message'] makes the printed error more clear?

Reply:
It's a bit too noisy imo.